Source/Core/InputCommon/ControllerEmu/ControlGroup/IMUGyroscope.h

```diff
@@ -23,7 +23,8 @@ class IMUGyroscope : public ControlGroup
   IMUGyroscope(std::string name, std::string ui_name);
 
   StateData GetRawState() const;
-  std::optional<StateData> GetState() const;
+  // Also updates the state
+  std::optional<StateData> GetState(bool update = false);
```

Comment:
Where is this called with `update = true`?  Assuming this was added for a future PR...?

Reply:
Good find, I should have changed the place that calls it to true like it's in the future PR. Forgot that.